app/models/ext_management_system.rb

```diff
@@ -449,8 +449,8 @@ def orchestrate_destroy
     disable! if enabled?
 
     if self.destroy == false
-      _log.info("Cant #{self.class.name} with id: #{id}, workers still in progress. Requeuing destroy...")
-      schedule_destroy_queue(id, :deliver_on => 15.seconds.from_now)
+      _log.info("Cant destroy #{self.class.name} with id: #{id}, workers still in progress. Requeuing destroy...")
```

Comment:
Can you change this to `Cannot destroy` or `Not destroying...`?   Sorry I didn't see this before.  `Cant` doesn't look right